frontend/src/pages/create_campaign/Roles/Question.js

```diff
@@ -11,26 +14,19 @@ const Question = ({
   onQuestionDelete,
 }) => (
   <>
-    <QuestionTitle>Question {questionNumber}</QuestionTitle>
-    <QuestionContent>
+    <Title>Question {questionNumber}</Title>
+    <Content>
       <TextField
         fullWidth
         multiline
         required
         value={question.text}
-        onChange={(e) => {
-          handleQuestionInput(e, question.id);
-        }}
+        onChange={(e) => handleQuestionInput(e, question.id)}
       />
-      <IconButton
-        value={question.id}
-        onClick={(e) => {
-          onQuestionDelete(e);
-        }}
-      >
+      <IconButton value={question.id} onClick={(e) => onQuestionDelete(e)}>
```

Comment:
Can we just use `onQuestionDelete` here instead of `(e) => onQuestionDelete(e)`?